src/libZilliqa/Zilliqa.cpp

```diff
@@ -148,6 +148,43 @@ Zilliqa::Zilliqa(const PairOfKey& key, const Peer& peer, SyncType syncType,
     LOG_STATE("[IDENT] " << string(key.second).substr(0, 8));
   }
 
+  // Logging useful information for seed nodes
+  if (syncType == SyncType::NEW_LOOKUP_SYNC) {
+    std::string startupInfo{"Node startup Information\n"};
+
+    // 3 mode: IP/Key/P2P Seed Key (ENABLE_SEED_TO_SEED_COMMUNICATION)
+    startupInfo += "Whitelisting mode   = ";
+    startupInfo += multiplierSyncMode
+                       ? "IP whitelisting mode\n"
+                       : ENABLE_SEED_TO_SEED_COMMUNICATION
+                             ? "Public Key (P2P Seed) whitelisting mode\n"
+                             : "Public Key only whitelisting mode\n";
+    // IP Address
+    startupInfo += "IP Address          = ";
+    startupInfo += peer.GetPrintableIPAddress();
+    startupInfo += "\n";
+
+    // External Seed Public Key
+    bytes extSeedPubKey;
+    extSeedKey.second.Serialize(extSeedPubKey, 0);
+    std::unique_ptr<char[]> extSeedPubKeyString;
+    Logger::GetPayloadS(extSeedPubKey, PUB_KEY_SIZE * 2, extSeedPubKeyString);
+    startupInfo += "ExtSeed Public Key  = ";
+    startupInfo +=
```

Comment:
can simply using just `std::string(extSeedKey.second)` ?

Reply:
Yes we can

```
ExtSeed Public Key  = 0x02CDDA9C5C40F6DF853B50E2CE3EBF7CB4CD66B8339685306B8A8A0140EE518576
```